backend/src/main/java/com/google/rolecall/services/SectionServices.java

```diff
@@ -117,17 +166,77 @@ public Section createSection(SectionInfo newSection) throws InvalidParameterExce
    */
   public Section editSection(SectionInfo newSection) throws EntityNotFoundException,
       InvalidParameterException {
+    Boolean isParentRevelation = newSection.type() == Section.Type.REVELATION;
+
+    System.out.printf("UPDATING %s\n", newSection.name());
```

Comment:
Please remove the printf here and other places in this class. If anything needs to be logged, please user logger.

Reply:
Done.

Reply:
Thanks.